src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -203,6 +204,7 @@ await using (Utf8JsonWriter writer = new(stream, new JsonWriterOptions() { Inden
                         writer.WriteEndArray(); // end parameterTypes
                         writer.WriteString("typeName", frame.TypeName);
                         writer.WriteString("moduleName", frame.ModuleName);
+                        writer.WriteString("moduleVersionId", frame.ModuleVersionId);
```

Comment:
Let's be explicit about how we serialize the mvid? e.g. `writer.WriteString("moduleVersionId", frame.ModuleVersionId.ToString("D"))`